notifier/main.ts

```diff
@@ -46,13 +46,17 @@ bot.chatType(["group", "supergroup"]).filter((ctx) =>
     await ctx.reply("Could not resolve PR branch.");
     return;
   }
+  const { file_path } = await ctx.api.getFile(ctx.message.document.file_id);
```

Comment:
What's wrong with using the files plugin here?

Reply:
For downloading it?

Reply:
For getting the URL

Reply:
Make a suggestion

Reply:
Did you find it?

Reply:
There's `file.getUrl`

Reply:
Oh, the URL. We can't use it for the URL.

Reply:
Why not?

Reply:
Because we don't use its URL here. We do in the `/file/:path` handler where we don't have grammY `ctx`.

Reply:
Right